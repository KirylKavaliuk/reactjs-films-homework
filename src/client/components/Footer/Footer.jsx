import React from 'react';

import styles from './Footer.scss';

const Footer = () => (
  <footer className={ styles.footer }>
    <h1 className={ styles.logo }> FILMS </h1>
    <span className={ styles.copy }>
      Copyright &copy; { new Date().getFullYear() } Kovalyuk Kirill
    </span>
  </footer>
);


export default Footer;
