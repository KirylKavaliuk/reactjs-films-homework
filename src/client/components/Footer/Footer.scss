@import 'styles/palette';

.footer {
  display: flex;
  flex-direction: column;
  width: 100%;
  background: $white;
  align-items: center;
  justify-content: center;
}

.logo {
  box-sizing: border-box;
  padding: 40px 10px 20px;
  color: $light_blue;
  font-size: 28px;
}

.copy {
  box-sizing: border-box;
  padding: 20px 10px;
  color: $grey;
}
