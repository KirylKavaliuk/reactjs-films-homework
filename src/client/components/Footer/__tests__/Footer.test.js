import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Footer from '../Footer';

it('renders footer shallow correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Footer/>);

  const tree = renderer.getRenderOutput();

  expect(tree).toMatchSnapshot();
});
