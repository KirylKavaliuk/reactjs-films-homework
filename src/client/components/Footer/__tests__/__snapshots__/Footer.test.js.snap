// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> 1`] = `
<footer
  className="Footer__footer___1WWdL"
>
  <h1
    className="Footer__logo___3Vf4V"
  >
     FILMS 
  </h1>
  <span
    className="Footer__copy___3S9MV"
  >
    Copyright Â© 
    2019
     Kovalyuk Kirill
  </span>
</footer>
`;
